### PR TITLE
Fix remove session

### PR DESCRIPTION
**Pull request by ashkarpetin**
Fixed remove session

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -352,7 +353,9 @@ public function send($method, $payload, $path, $params = [], array $headers = []
         $response = call_user_func($this->middleware, $request, $response, $this);
 
         if ($response->getStatusCode() === 404) {
-            throw new Http\Exception\NotFoundException();
+            $content = json_decode($response->getBody()->getContents(), true);
+
+            throw new Http\Exception\NotFoundException($content['message'] ?? '');
```
Comment (VladimirBerdnik) [inline src/Client.php]: Can't we remove FQCN? I see that it is imported

Comment (ashkarpetin) [reply src/Client.php]: I've accidentally imported it. Removed import.

File: src/Services/SessionService.php [hunk 2]
```diff
@@ -67,11 +67,8 @@ public function update($tokenId, $data)
...
     }
 
-    /**
-     * @param string $tokenId
-     */
-    public function delete($tokenId)
+    public function logout()
     {
-        $this->client()->delete('sessions/{tokenId}', ['tokenId' => $tokenId]);
+        return $this->client()->post([], 'logout');
```
Comment (VladimirBerdnik) [inline src/Services/SessionService.php]: Do we expect some data in response? Is it possible to add PHPDoc?

**Review by VladimirBerdnik — APPROVED** [review]

File: src/Client.php [hunk 3]
```diff
@@ -352,7 +352,9 @@ public function send($method, $payload, $path, $params = [], array $headers = []
...
```
Comment (slavcodev) [inline src/Client.php]: I think this is incorrect. The exception message is in the HTTP Status reason

Comment (ashkarpetin) [reply src/Client.php]: `$response->getReasonPhrase()` returns 'Not Found'. I need detailed error: 'Customer not found'. How can I achieve that?

Comment (slavcodev) [reply src/Client.php]: Improve message on backend?

**Review by slavcodev — APPROVED** [review]

**Pull request merged**